app/src/layouts/appLayout/appLayout.jsx

```diff
@@ -21,14 +21,9 @@
 
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import { ScrollWrapper } from 'components/main/scrollWrapper';
-import { Sidebar } from './sidebar';
-import { Header } from './header';
-import { Footer } from './footer';
-import styles from './appLayout.scss';
-
-const cx = classNames.bind(styles);
+import { Layout } from 'layouts/common/layout';
+import { AppHeader } from './appHeader';
+import { AppSidebar } from './appSidebar';
 
 export class AppLayout extends PureComponent {
```

Comment:
Could you please make it a functional component too?

Reply:
Done